data/json/npcs/godco/NPC_Aunt_Theresa.json

```
@@ -0,0 +1,239 @@
+          "and": [
+            { "npc_has_var": "u_want_blanket", "type": "general", "context": "trade" },
+            {
+              "npc_compare_time_since_var": "knitting",
+              "type": "timer",
+              "context": "trade",
+              "op": ">=",
+              "time": "2 d"
+            }
+          ]
+        },
+        "effect": [ { "u_spawn_item": "blanket" } ],
+        "topic": "TALK_GODCO_Theresa_Knit_Done"
+      },
+      {
+        "text": "I'm here about the quilt.",
+        "condition": {
+          "and": [
+            { "npc_has_var": "u_want_quilt", "type": "general", "context": "trade" },
+            {
+              "npc_compare_time_since_var": "knitting",
+              "type": "timer",
+              "context": "trade",
+              "op": ">=",
+              "time": "7 d"
+            }
+          ]
+        },
+        "effect": [ { "u_spawn_item": "quilt" } ],
+        "topic": "TALK_GODCO_Theresa_Knit_Done"
+      },
+      { "text": "Well, it's been nice talking to you, but I need to head out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Theresa_Mood",
+    "dynamic_line": "We're doing better than most, to put it mildly.  There's a reason for that, as for everything.  Say, have I told you about my husband?  I'm sorry if I already have, my memory's not what it used to be.",
+    "responses": [
+      { "text": "Tell me about your husband.", "topic": "TALK_GODCO_Theresa_Husband" },
+      { "text": "Well, it's been nice talking to you, but I need to head out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Theresa_Husband",
+    "dynamic_line": "He was an exceptional man.  Ever since he was a child it was clear that there was something special about him.  My husband had a way with people, and with God too.  But, after all he was as human as you and I.  He was corrupted by addiction.  One day, God appeared to him in a vision that turned his life around, he had other plans for him.  From then on he didn't take his life for granted.  He used his talent and newfound inspiration to better the people around him.",
+    "responses": [
+      { "text": "So, he founded the community?", "topic": "TALK_GODCO_Theresa_Community" },
+      { "text": "Tell me about his visions.", "topic": "TALK_GODCO_Theresa_Vision" },
+      { "text": "Well, it's been nice talking to you, but I need to head out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Theresa_Community",
+    "dynamic_line": "That's right.  He wanted something to return for the guidance he received when he was at the lowest point of his life.  He created a charity group which, over the years, became what's now known as the New England Church Community.  He lead them for many decades, until his death a few years before his prophecy was fulfilled.  Now, his niece Helena leads the community.",
```

```suggestion
    "dynamic_line": "That's right.  He wanted something to return for the guidance he received when he was at the lowest point of his life.  He created a charity group which, over the years, became what's now known as the New England Church Community.  He lead them for many decades, until his death a few years before his prophecy was fulfilled.  Now, our niece Helena leads the community.",
```

Wouldn't Helena be both of their niece?

---

Yes, she would.